relay/sources/relayd/Cargo.toml

```diff
@@ -7,7 +7,7 @@ description = "Rudder relay component"
 readme = "README.md"
 homepage = "https://www.rudder.io"
 repository = "https://github.com/Normation/rudder"
-license = "GPL3"
+license = "GPL-3.0-or-later"
```

Comment:
Cargo actually expects an SPDX-formatted license name, things are converging somehow.

Reply:
but or-later is not the sale a gpl-3

Reply:
Indeed, and previous value was actually wrong (https://github.com/Normation/rudder/blob/master/webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/api/DataStructures.scala#L10 or any GPL header in Rudder).